src/NUnitFramework/nunit.framework.legacy.tests/AssertEqualsTests.cs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
 
+#pragma warning disable NUnit2005
+
```

Comment:
This is already excluded by the .editorconfig you added so manual pragma's are not needed.
We should also delete the pre-existing `#pragma` then from this file.